md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ProcessAPI.java

```diff
@@ -201,7 +201,7 @@ RestWrapper list(@RequestParam(value = "page", defaultValue = "0") int startPage
             restWrapper = new RestWrapper(processes, RestWrapper.OK);
             LOGGER.info("All records listed from Process by User:" + principal.getName());
         } catch (Exception e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1166)